llvm/docs/ReleaseNotes.md

```diff
@@ -76,6 +76,10 @@ Changes to the LLVM IR
   guaranteed to start with a `bti` or `endbr64` instruction, where
   those exist.
 
+* The `ptrtoaddr` instruction was introduced. This instruction returns the
+  address component of a pointer type variable but unlike `ptrtoint` does not
+  capture provenance ([#125687](https://github.com/llvm/llvm-project/pull/125687)).
+
```

Comment:
@tru For the 21.0.git bump this file was reset to empty (https://github.com/llvm/llvm-project/commit/3bd3e06f3fe418e24af65457877f40cee0544f9d), but it seems this did not happen for the 22.0 commit: https://github.com/llvm/llvm-project/commit/01f36b39bd2475a271bbeb95fb9db8ed65e2d065